tests/test_main.py

```diff
@@ -337,3 +337,25 @@ def test_veracode_main_no_sandbox_name(
             ),
         ):
             self.assertIsNone(main.main())
+
+        # Test for UnboundLocalError
```

Comment:
Note that if you add this test to `master` it will fail, and with the rest of the changes in this PR it will succeed.